src/encoded/static/components/matrix_chip_seq.js

```diff
@@ -651,6 +656,12 @@ class ChIPSeqMatrixPresentation extends React.Component {
             storedSelectedTabLevel3 :
             this.subTabs.length > 0 ? this.ChIPSeqMatrixData.subTabs[0] : null;
 
+        // Note: Hacky. If assay title is Histone and the organism is worm or fly, direct the user to TF assay title
+        if (assayTitle === 'Histone ChIP-seq' && organismsWithHiddenHistoneAssay.includes(organismName)) {
+            const tfUrl = link.replace('assay_title=Histone%20ChIP-seq', 'assay_title=TF ChIP-seq').replace('assay_title=Histone ChIP-seq', 'assay_title=TF ChIP-seq');
```

Comment:
While this works, would it be cleaner to use the query_string library, or an unneeded complication?